.github/workflows/quick-start.yml

```diff
@@ -0,0 +1,47 @@
+name: quick-start
+
+on:
+  workflow_call:
+    inputs:
+      fixtures:
+        required: false
+        type: string
+      
+      superuser:
+        required: false
+        type: string
+
+      web_service:
+        required: false
+        type: string
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Start docker containers
+        run: docker compose up -d --build || ( docker compose logs >&2 && exit 1; )
+      - name: Wait for migrations to finish
+        run: |
+          echo "Waiting for migrations to complete..."
+          until ! docker compose exec -T web src/manage.py showmigrations | grep -q '\[ \]'; do
+            echo "Migrations not finished, waiting..."
+            sleep 3
+          done
+      - name: Load fixtures
+        if: ${{ inputs.fixtures != '' }}
+        run: docker compose exec -T web src/manage.py loaddata ${{ inputs.fixtures }}
+      - name: Create superuser
+        run: |
+          SUPERUSER="${{ inputs.superuser || 'admin' }}"
+          WEB_SERVICE="${{ inputs.web_service || 'web' }}"
+          docker compose exec -T $WEB_SERVICE src/manage.py createsuperuser --username $SUPERUSER --email $SUPERUSER@admin.nl --no-input
+      - name: Check main page
+        run: |
+          curl_status=$(curl -w '%{http_code}' -o /dev/null  -s http://localhost:8000/)
```

Comment:
I think it might be good to make the port number variable (with `8000` as the default), currently all of them are probably `8000`, but if we introduce any other numbers it's good to have this prepared for that